lib/libmbedtls/core/tee_lmd_provider.c

```diff
@@ -75,6 +85,110 @@ static int mbd_rand(void *rng_state, unsigned char *output, size_t len)
 	* defined(CFG_MBEDTLS_HMAC_PRNG)
 	*/
 
+#if defined(_CFG_CRYPTO_WITH_CIPHER) || defined(_CFG_CRYPTO_WITH_MAC) || \
+	defined(_CFG_CRYPTO_WITH_AUTHENC)
+/*
+ * Get the Mbedtls chipher info given a TEE Algorithm "algo"
+ * Return
+ * - TEE_SUCCESS in case of success,
```

Comment:
A pointer to a valid `mbedtls_cipher_definition_t` on success

Reply:
Yes, I will fix it.